torch/nn/_intrinsic/quantized/modules/conv_relu.py

```diff
@@ -24,29 +24,13 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
                                          stride=stride, padding=padding, dilation=dilation,
                                          groups=groups, bias=bias, padding_mode=padding_mode)
 
-    def weight(self):
```

Comment:
Are these two methods being removed?

Reply:
Yes, I don't see why they are needed as the parent module already defined them. Looks like linear_relu.py also doesn't have them